interface/ceed-operator.c

```diff
@@ -907,6 +910,29 @@ int CeedCompositeOperatorAddSub(CeedOperator composite_op,
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Check if CeedOperator setup is complete.
+
+  Note: A successful error code indicates that the CeedOperator
+           has been set as immutable.
+
+  @param op        CeedOperator
+  @param is_ready  Boolean flag indicating if operator setup is complete
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Advanced
+**/
+int CeedOperatorIsReady(CeedOperator op, bool *is_ready) {
+  int ierr;
+
+  *is_ready = false;
+  ierr = CeedOperatorCheckReady(op); CeedChk(ierr);
+  *is_ready = true;
+
+  return CEED_ERROR_SUCCESS;
```

Comment:
These are surprising semantics. One would expect to return with `is_ready=false` and no error code. `CeedOperatorCheckReady` provides more information than a bool (in the form of an error code). If we want that to be available in a non-error path, then perhaps the innermost function should be this non-error check. I'm skeptical that it's worth doing that -- a caller can just handle the error.